src/components/App.js

```diff
@@ -1,12 +1,10 @@
 import DisplayCalculator from './DisplayCalculator';
 import ButtonPanel from './ButtonPanel';
-import Calculate from '../logic/calculate';
```

Comment:
Oops, you're not supposed to remove this line, you will need it in the next milestone, kindly add it back before merging, I trust you can do this. 👀  on you!....